olp-cpp-sdk-dataservice-read/tests/CatalogRepositoryTest.cpp

```diff
@@ -28,35 +28,47 @@
+constexpr auto
+    kResponseConfig =
```

Comment:
Please format.

Reply:
Do you mean that it begins from new line? It is actually how clang formats it.